plugin.json

```diff
@@ -31,6 +31,13 @@
         "help_text": "Time of day to run the Legal Hold task, in the form 'HH:MM ±HHMM' (e.g. '3:00am -0700').  Use +0000 for UTC.",
         "default": "1:00am -0700"
       },
+      {
+        "key": "EnableFilestoreConnectionTest",
```

Comment:
I think this should be "DisableFilestoreConnectionTest" to better emphasize this test should normally be active.

Reply:
Yes, not a big issue.